phpunit.xml.dist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Use this configuration file as a template to run the tests against any dbms.
+    Procedure:
+        1) Save a copy of this file with a name of your choosing. It doesn't matter
+           where you place it as long as you know where it is.
+           i.e. "mysqlconf.xml" (It needs the ending .xml).
+        2) Edit the file and fill in your settings (database name, type, username, etc.)
+           Just change the "value"s, not the names of the var elements.
+        3) To run the tests against the database type the following from within the
+           tests/ folder: phpunit -c <filename> ...
+           Example: phpunit -c mysqlconf.xml AllTests
+-->
```

Comment:
Probably needs to be removed since this repo does not have to do with dbms'es

Reply:
True. I guess we don't need the hint to copy the dist-file into a xml either, so I removed it.